photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/training/Driver.scala

```diff
@@ -160,6 +171,28 @@ final class Driver(val sc: SparkContext, val params: GameParams, implicit val lo
     }
 
   /**
+   * Run various checks on data (training or validation. Throws a RuntimeException if any of the checks does not pass.
+   *
+   * @note this is somewhat expensive, so we can turn it off
+   *
+   * Checks performed
+   * ================
+   * - all weights >= 0.0 and at least 1 weight > 0
+   *
+   * @param data The data to check
+   */
+  protected[training] def checkData(data: DataFrame): DataFrame =
+
+    if (params.checkData)
```

Comment:
The databricks style guide requires braces unless you can fit the entire expression on one line:

https://github.com/databricks/scala-style-guide#curly